pkg/converter/sloppy_file_test.go

```diff
@@ -0,0 +1,86 @@
+package converter_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/google/go-cmp/cmp"
+	"github.com/sloppyio/sloppose/pkg/converter"
+)
+
+var expectedSloppyYml = `project: pkg
+services:
+  apps:
+    busy_env:
+      cmd: sleep 20
+      dependencies:
+      - ../apps/db
+      - ../apps/wordpress
+      domain: $URI
+      env:
+      - VAR_A=1
+      - VAR_B=test
+      image: busybox
+      instances: 1
+      mem: 256
+    db:
+      cmd: mysqld
+      domain: $URI
+      env:
+      - MYSQL_DATABASE=wordpress
+      - MYSQL_PASSWORD=wordpress
+      - MYSQL_ROOT_PASSWORD=somewordpress
+      - MYSQL_USER=wordpress
+      image: mysql:8.0.0"
+      instances: 1
+      mem: 256
```

Comment:
Similar to volume size, instances and mem also have defaults, so there is no need to generate those.

Though it would be good to add memory to one of the services in the input file and match it here.